src/a11y-dialog.ts

```diff
@@ -179,15 +179,10 @@ export default class A11yDialog {
    * (namely ESC and TAB)
    */
   private bindKeypress = (event: KeyboardEvent) => {
-    // This is an escape hatch in case there are nested dialogs,
-    // so the keypresses are only reacted to for the most recent one
-    if (!this.$el.contains(document.activeElement)) return
-
```

Comment:
🔍 This is no longer necessary if we bind this event listener specifically to `this.%el`, instead of to the document

Reply:
Can you double check with the nested dialogs fixture that it still works okay?

Reply:
It does seem to work ok. I did some manual testing and Cypress is still happy.